packages/browser/README.md

```diff
@@ -217,6 +217,73 @@ htevents.register(lowercase)
 
 For further examples check out our [existing plugins](/packages/browser/src/plugins).
 
+# Client-side destinations
+
+The Browser SDK supports sending events directly from the client to destinations which is useful in situations where the destination requires a client-side context in order to fully enrich and attribute events.
+
+## Google Analytics 4
+
+Google Analytics 4 (GA4) offers tracking via Google Tag Manager (GTM) which may benefit from a client-side integration.
+
+### Installation
+
+Make sure your GA4 setup scripts are configured on your website. Our implementation expects the `gtag` function to be available in the global scope.
+
+```html
+<!-- example GA4 setup using Google Tag Manager -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-XXXXXXXX"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag() { dataLayer.push(arguments); }
+  gtag('js', new Date());
+  gtag('config', 'G-XXXXXXXX');
+</script>
+```
+
+You can then configure the Browser SDK to send events directly to GA4 by enabling the `Google Tag Manager` destination:
+
+```js
+htevents.load("WRITE_KEY", {
+  destinations: {
+    "Google Tag Manager": {
+      measurementId: "G-XXXXXXXX"
+    }
+  }
+})
+```
+
+View the complete plugin documentation in [`google-tag-manager.ts`](src/plugins/destinations/google-tag-manager.ts#L11)
+
+### Usage
+
+Once the destination is configured, all applicable `identify`, `track`, and `page` events will be sent. The integration also automatically populates the `user_id` and `hightouch_anonymous_id` fields.
```

Comment:
> `user_id` and `hightouch_anonymous_id` fields

Is this precedent from seg? Does the original integration set `user_id` and `segment_anonymous_id`?

Reply:
Yes, Segment auto-populates the `userId` and `segmentAnonymousId` fields. I changed to snake_case since that is the standard in GA4.